### DISCUSSION TITLE
is this just flamework but in lua?

### DISCUSSION THREAD

**Discussion by plac3** [welcomestohell/prvdmwrong#29] [Q&A] [answered]
the internal code looks the exact same

**Comment by rniraclefire** [comment, marked as answer]
Prvd 'M Wrong is inspired by Flamework and made from my grudges with Flamework being too coupled :)

**Comment by plac3** [comment]
I mean I wouldn't call a direct transpile "inspiration" considering parts
of the code are the exact same except slightly modified to look less like a
blatant copy...

**Reply by rniraclefire** [reply]
Zero transpiling was used. The most that was needed was Flamework's reflect module and its topological sort. That is it.

Prvd 'M Wrong will diverge from Flamework significantly. I do not consider it a Luau variant of Flamework, just a game framework I made from grudges with existing frameworks.

Again, if you think it's a blatant copy, consider filing an issue to see what we can do better :)
